internal/backends/winit/renderer/skia.rs

```diff
@@ -17,22 +17,16 @@ mod cached_image;
 mod itemrenderer;
 mod textlayout;
 
-#[cfg(not(any(target_os = "macos", target_family = "windows")))]
-mod opengl_surface;
-
-#[cfg(target_os = "macos")]
-mod metal_surface;
-
-#[cfg(target_family = "windows")]
-mod d3d_surface;
-
 cfg_if::cfg_if! {
-    if #[cfg(target_os = "macos")] {
+    if #[cfg(skia_backend_opengl)] {
+        mod opengl_surface;
+        type DefaultSurface = opengl_surface::OpenGLSurface;
+    } else if #[cfg(skia_backend_metal)] {
+        mod metal_surface;
         type DefaultSurface = metal_surface::MetalSurface;
-    } else if #[cfg(target_family = "windows")] {
+    } else if #[cfg(skia_backend_d3d)] {
+        mod d3d_surface;
         type DefaultSurface = d3d_surface::D3DSurface;
-    } else {
```

Comment:
and else?

Reply:
There's no else :). I mean, that should produce a compile error - would happen if a new renderer was added (say Vulkan) starting with build.rs.

Reply:
What happens on linux if the renderer-skia is set but not renderer-skia-opengl ?

Reply:
That shouldn't happen. Ah, it can because of a bug in build.rs - fix pushed.